tests/integration/utils.py

```diff
@@ -274,27 +274,6 @@ def make_list_of_packages_hashable(data):
     return sorted([[i["name"], i["type"], i["version"]] for i in data])
 
 
-def sort_pkgs_and_deps_in_place(packages=None, dependencies=None):
```

Comment:
I don't think it's a good idea to remove the function completely. It can be helpful for creating new tests / changing existing ones.

Reply:
@chmeliik @ejegrova  WDYT?

Reply:
I don't think we'll need it for creating new tests anymore. The Cachito API will now always return the packages and dependencies properly sorted, so it should be easy to just copy-paste the content when creating the test data